pkg/apic/client.go

```diff
@@ -274,7 +276,23 @@ func (c *ServiceClient) checkAPIServerHealth() error {
 		// Set the team Id
 		c.cfg.SetTeamID(team.ID)
 	}
-	return nil
+
+	// reset the cache of team names
+	return c.setTeamCache()
+}
+
+func (c *ServiceClient) setTeamCache() error {
+	// passing nil to getTeam will return the full list of teams
+	platformTeams, err := c.getTeam(nil)
```

Comment:
Would making it map[string]string{} return a full as well?

Reply:
probably. I didn't try that one. Should I change it?

Reply:
I dunno... what's the stance of sending in nil vs. an instantiated empty?

Reply:
probably only matters if the func being called handles them correctly. If it didn't handle nil, it wouldn't have worked and I wouldn't have done it that way.

Reply:
I think Shane is right, you can send in the empty map as it would result in no query params just the same.

Reply:
I will change it, but I've found other examples (e.g. GetProxyList in pollfrontendproxyjob.go, among others) that also are passing in nil for the query params rather than an empty map.